cloudsync_onedrive.py

```diff
@@ -1092,6 +1113,10 @@ def _info_from_rest(self, item, root=None):
         name = item["name"]
         if root:
             path = self.join(root, name)
+            path_orig = path
```

Comment:
It's a nit, and also not exactly your code, but I'm not a fan of initializing variables in a conditional? `if not root: raise NotImplementedError()` makes the code path a bit clearer, I think

Reply:
good point, made the adjustment